tools/get.py

```diff
@@ -71,13 +71,13 @@ def get_tool(tool):
     local_path = dist_dir + archive_name
     url = tool['url']
     #real_hash = tool['checksum'].split(':')[1]
-    if 'CYGWIN_NT' in sys_name:
+    if 'CYGWIN_NT' not in sys_name:
```

Comment:
Ah I missunderstood what was going on here, I assumed the starting state before https://github.com/espressif/arduino-esp32/commit/4865ed0d1382f67d7ed482edc20b81ce6835c0bb was "ssl is enabled" but I see now you are correct it was disabled beforehand and that commit made it disabled only for windows. My mistake, sorry.